demisto_sdk/commands/content_graph/objects/base_content.py

```diff
@@ -78,6 +78,8 @@ class BaseContent(ABC, BaseModel, metaclass=BaseContentMetaclass):
     content_type: ClassVar[ContentType] = Field(include=True)
     node_id: str
     marketplaces: List[MarketplaceVersions] = list(MarketplaceVersions)
+    not_in_repository: bool = False
```

Comment:
Why do we need this field?

Reply:
This field already existed in the UnknownContent class, so in order for me to be able to use this field in both of the classes that inherit from BaseContent (which is UnknownContent and ContentItem) @ilaner and I decided to add this field to the BaseContent class.